_includes/releases/v22.1/v22.1.0-beta.5.md

```diff
@@ -0,0 +1,51 @@
+## v22.1.0-beta.5
+
+Release Date: May 3, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-0-beta-5-security-updates">Security updates</h3>
+
+- `Crdb_internal.reset_sql_stats()` and `crdb_internal.reset_index_usage_stats()` builtins now check if user has the admin role. [#80384][#80384]
+- [#80248][#80248] [7d55af0e6][7d55af0e6]  (Raphael 'kena' Poss <knz@cockroachlabs.com>)
+
+<h3 id="v22-1-0-beta-5-enterprise-edition-changes">Enterprise edition changes</h3>
+
+- Backups run by secondary tenants now write protected timestamp records to protect their target schema objects from garbage collection during backup execution. [#80670][#80670]
```

Comment:
"Backups" link to take-full-and-incremental-backups.html 

Also not 100% that this note needs to be included in the release notes. If you had enough time, I would double check with engineer.